lib/Core/Executor.cpp

```diff
@@ -1476,7 +1476,7 @@ Executor::StatePair Executor::branchFork(ExecutionState &current,
       // case also we extract the unsat core of the proof
       txTree->markPathCondition(current, unsatCore);
       if (WPInterpolant)
-        txTree->markInstruction(current.prevPC, true);
+        txTree->markInstruction(current.prevPC, false);
```

Comment:
May I ask why we have changed the second argument from false to true?

Reply:
Because if res == Solver::True we mark 1, res == Solver::False we mark 2